doc/guides/device_mgmt/mcumgr_backporting.rst

```diff
@@ -0,0 +1,161 @@
+.. _management_fixing_prev_mcumgr:
+
+Fixing and backporting fixes to Zephyr v2.7 MCUmgr
+##################################################
+
+The processes described in this document apply to both the `upstream Zephyr repository <https://github.com/zephyrproject-rtos/zephyr>`_
+and the MCUmgr repository pointed by the ``west.yml`` file in Zephyr.
+
+.. note::
+   Currently, only Zephyr version 2.7 LTS requires the backporting of the Zephyr documentation.
```

Comment:
What is "the backporting of the Zephyr documentation"? Reading this note at this point in the document doesn't make sense to me. Some context introducing what this page is for would be helpful.

Reply:
I have just badly written:
"Currently the backporting process, described in this document, is required only when providing changes to Zephyr version 2.7 LTS"
or any sentence that means: we only do that for 2.7 right now.